packages/core/src/sass/app/_common/components/app-notification-message.scss

```diff
@@ -12,8 +12,7 @@
     }
 
     @include mobile {
-        left: 1.6rem;
-        right: 1.6rem;
+        width: 100%;
```

Comment:
Wouldn't this affect existing app notification messages on `mobile`?

Reply:
Yep that's right, the original styling was going out of the frame on the right side. I just pushed up another commit to ensure consistency between the sizes of the notifications.

Reply:
As discussed, can we use `max-width` with offset calculation instead? 🙏🏻

Reply:
```suggestion
        right: 1.6rem;
        max-width: calc(100% - 3.2rem);
```

Reply:
Sure